lib/travis/build/script/go.rb

```diff
@@ -103,6 +108,13 @@ def uses_make
             '-f GNUmakefile || -f makefile || -f Makefile || -f BSDmakefile'
           end
 
+	  def uses_15_vendoring
+            if (go_version < '1.5' || go_version == 'go1')
```

Comment:
There's also `Gem::Version` as explained in http://stackoverflow.com/a/3064161/433558 that's likely viable.